pkg/k8shandler/trustedcabundle.go

```diff
@@ -0,0 +1,65 @@
+package k8shandler
+
+import (
+	"fmt"
+	"hash/fnv"
+	"strconv"
+
+	"github.com/openshift/cluster-logging-operator/pkg/utils"
+	"github.com/sirupsen/logrus"
+	core "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+)
+
+/*
+ * Create or update Trusted CA Bundle ConfigMap
+ * By setting label "config.openshift.io/inject-trusted-cabundle: true", the cert is automatically filled/updated.
+ */
+func (clusterRequest *ClusterLoggingRequest) createOrUpdateTrustedCABundleConfigMap(configMapName string) error {
+	logrus.Debug("createOrUpdateTrustedCABundlenfigMap...")
+	labels := map[string]string{
+		injectTrustedCABundleLabel: "true",
+	}
+	configMap := NewConfigMapWithLabels(
+		configMapName,
+		clusterRequest.cluster.Namespace,
+		labels,
+		map[string]string{
+			trustedCABundleKey: "",
+		},
+	)
+
+	utils.AddOwnerRefToObject(configMap, utils.AsOwner(clusterRequest.cluster))
+
+	err := clusterRequest.Create(configMap)
+	if err != nil && !errors.IsAlreadyExists(err) {
+		return fmt.Errorf("Failure constructing trusted CA bundle configmap: %v", err)
+	}
+	return nil
+}
+
+func hasTrustedCABundle(configMap *core.ConfigMap) bool {
+	if configMap == nil {
+		return false
+	}
+	caBundle, ok := configMap.Data[trustedCABundleKey]
+	if ok && caBundle != "" {
+		return true
+	} else {
+		return false
+	}
+}
+
+func calcTrustedCAHashValue(configMap *core.ConfigMap) string {
```

Comment:
Is it possible to use: https://github.com/openshift/cluster-logging-operator/blob/master/pkg/utils/utils.go#L59